js/main.js

```diff
@@ -52,3 +55,214 @@ function appendPokemonDataForSelect(event) {
+    $p3.textContent = 'Attack: ' + data.currentPokemon.attack;
+
+    $divRow3.appendChild($p3);
+
+    var $p4 = document.createElement('p');
+    $p4.textContent = 'Defense: ' + data.currentPokemon.defense;
+    $divRow3.appendChild($p4);
+    var $p5 = document.createElement('p');
+    $p5.textContent = 'Special-defense: ' + data.currentPokemon.specialDefense;
+
+    $divRow3.appendChild($p5);
+    var $p6 = document.createElement('p');
+
+    $p6.textContent = 'Special-attack: ' + data.currentPokemon.specialAttack;
+    $divRow3.appendChild($p6);
+
+    var $p7 = document.createElement('p');
+    $p7.textContent = 'Speed: ' + data.currentPokemon.speed;
+
+    var $p8 = document.createElement('p');
+    $p8.textContent = 'Weight: ' + data.currentPokemon.weight;
+
+    var $p9 = document.createElement('p');
+    $p9.textContent = 'Height: ' + data.currentPokemon.height;
+
+    $divRow3.appendChild($p7);
+    $divRow3.appendChild($p8);
+    $divRow3.appendChild($p9);
+
+    $divColumn2.appendChild($divRow3);
+
+    var $divColumn3 = document.createElement('div');
+    $divColumn3.setAttribute('class', 'column-half column-full display-flex align-center justify-center');
+
+    $pokemonDetailsBorder.appendChild($divColumn3);
+
+    var $officialArt = document.createElement('img');
+    $officialArt.setAttribute('class', 'official-artwork padding-top-small');
+    $officialArt.setAttribute('src', data.currentPokemon.img);
+
+    $divColumn3.appendChild($officialArt);
+    $divRow2.appendChild($divColumn3);
+
+    $pokemonDetail.appendChild($pokemonDetailsBorder);
+
+  });
+  xhr.send();
+}
+
+/* <div class="pokemon-details-border height-test">
+  <div class="row">
+    <div class="column-full pokemon-details-header padding-top-small">
+      <div class="row padding-left-25">
+        <h1>#1 Bulbasaur フシギダネ</h1>
+      </div>
+    </div>
+  </div>
+  <div class="row flex-wrap">
+    <div class="column-half column-full details">
+      <p>A strange seed was\nplanted on its\nback at birth.\fThe plant sprouts\nand grows with\nthis POKéMON.</p>
+      <button class="grass-type">GRASS</button>
+      <button class="poison-type">POISON</button>
+      <div class="stats padding-top-small">
+        <p class="bot-margin-none">HP: 99</p>
+        <p class="bot-margin-none">Attack: 99</p>
+        <p class="bot-margin-none">Special-attack: 99</p>
+        <p class="bot-margin-none">Special-defense: 99</p>
+        <p class="bot-margin-none">Defense: 99</p>
+        <p class="bot-margin-none">Speed: 99</p>
+        <p class="bot-margin-none">Weight: 99</p>
+        <p class="bot-margin-none">Height: 99</p>
+      </div>
+    </div>
+    <div class="column-half column-full display-flex align-center justify-center">
+      <img class="official-artwork padding-top-small" src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/1.png" alt="bulbasaur">
+    </div>
+  </div>
+</div> */
```

Comment:
Make sure this is inside the function block for the function that generates this DOM tree